test/recurly/BaseClient.test.js

```diff
@@ -19,7 +19,7 @@ describe('BaseClient', () => {
   describe('#constructor', () => {
     it('Should set the internal state and headers', () => {
       assert.equal(client._getDefaultOptions().headers['Authorization'], 'Basic bXlhcGlrZXk6')
-      assert.equal(client._getDefaultOptions().headers['User-Agent'], `Recurly/${pkg.version}; node`)
+      assert.equal(client._getDefaultOptions().headers['User-Agent'], `Recurly/${pkg.version}; node ${process.version}`)
```

Comment:
It might be better to use a regex here. I feel like this is testing the implementation. Will address in a follow up.

Reply:
I made a backlog ticket for this.